os/Kconfig.debug

```diff
@@ -77,6 +77,38 @@ config DEBUG_AUDIO_INFO
 
 endif #DEBUG_AUDIO
 
+if LCD
+
+config DEBUG_LCD_ERROR
+	bool "Enable LCD Error Debug Output"
+	default n
+	depends on DEBUG_ERROR
+	---help---
+		LCD Error Debug Output.
+
+config DEBUG_LCD_INFO
+	bool "Enable LCD Informational Debug Output"
+	default n
+	depends on DEBUG_VERBOSE
+	---help---
+		LCD Informational Debug Output.
+
+config DEBUG_GRAPHICS_ERROR
+	bool "Enable Graphics Error Debug Output"
+	default n
+	depends on DEBUG_ERROR
+	---help---
+		Graphics Error Debug Output.
+
+config DEBUG_GRAPHICS_INFO
+	bool "Enable Graphics Informational Debug Output"
+	default n
+	depends on DEBUG_VERBOSE
+	---help---
+		Graphics Informational Debug Output.
+
+endif #DEBUG_LCD
```

Comment:
Line 80 using  "LCD" does not match here comment "DEBUG_LCD"

Reply:
LCD is not match the comment DEBUG_LCD,

Reply:
fixed, thanks